### PR TITLE
feat(DTFS2-5215): Portal deal exporter sorting

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:
* Added ability to sort deals by exporter.  It was handled by component but that only sorted the page, not all deals.  Now all exporters are sorted alphabetically
* Added e2e test

### CODE REVIEW

File: e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-sort.spec.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  BSS_DEAL_MIA,
+  GEF_DEAL_DRAFT,
+} = require('../fixtures');
+
+const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
+const chance = new Chance();
+
+context('Dashboard Deals filters - filter by multiple fields with multiple values', () => {
+  const ALL_DEALS = [];
+  const asda = { ...GEF_DEAL_DRAFT };
```
Comment (IgnatG) [inline e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-sort.spec.js:16]: can we change the name for this one? I forgot to do it when I pushed my changes

Comment (Zainzzkk) [reply e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-sort.spec.js]: done

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Good work.

File: e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-deal-with-issued-facilities/submit-deal-with-issued-facilities.spec.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ context('Portal to TFM deal submission', () => {
       const valueinGBP = value.text().split(' ');
       // removes commas
       const exposureValue = parseFloat(valueinGBP[1].replace(/,/g, '')) * (issuedBond.coveredPercentage / 100);
-      facilityRow.facilityExposure().contains(`GBP ${exposureValue.toFixed(2)}`);
+      facilityRow.facilityExposure().contains(`GBP ${exposureValue.toFixed(0)}`);
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-deal-with-issued-facilities/submit-deal-with-issued-facilities.spec.js]: done

**Review by abhi-markan — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**